Composer/packages/client/src/components/InstallDepModal.tsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable react/no-danger */
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { DialogTypes, DialogWrapper } from '@bfc/ui-shared/lib/components/DialogWrapper';
+import { jsx } from '@emotion/core';
+import formatMessage from 'format-message';
+import { DefaultButton, PrimaryButton } from 'office-ui-fabric-react/lib/components/Button';
+import { DialogFooter } from 'office-ui-fabric-react/lib/components/Dialog';
+import React from 'react';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+
+const dialogFooterClass = mergeStyles({
```

Comment:
dont need to use mergeStyles, just directly assign marginTop: 25px to the root under styles.
<DialogFooter styles={{ root: { marginTop: 25 }}} .../>

Reply:
Ah, weird. Dialog Footer doesn't have a root style, only styles for the buttons. I think this may have been done as a workaround for that.  

I can apply the css directly to the element.